config/routes.rb

```diff
@@ -22,13 +22,14 @@
   end
 
   post '/Patient/:patient_id/$health-cards-issue', to: 'health_cards#create', as: :issue_vc, format: :fhir_json
-  get "/Patient/:id", to: "patients#show", as: :fhir_patient, format: :fhir_json
-  get "/Immunization/:id", to: "immunizations#show", as: :fhir_immunization, format: :fhir_json
-  get "/.well-known/smart-configuration", to: "well_known#smart", as: :well_known_smart, format: :json
-  get "/.well-known/jwks", to: "well_known#jwks", as: :well_known_jwks, format: :json
-  get "/metadata", to: "metadata#capability_statement", as: :fhir_capabilitystatement, format: :fhir_json
-  get "/OperationDefinition/health-cards-issue", to: "metadata#operation_definition", as: :fhir_operationdefinition, format: :fhir_json
+  get '/Patient/:id', to: 'patients#show', as: :fhir_patient, format: :fhir_json
```

Comment:
I was curious why this wasn't caught by rubocop, and it looks like we're excluding the entire config directory. Should we re-enable that directory?

Reply:
I wouldn't because the majority of those files are generated.